docs: updates RELEASES.md (#1)
mathetake/ai-gateway: RELEASES.md

# Notes on Releases

## Cycles

> TODO

Author: 🥸

# Notes on Releases

## Cycles

Since the AI space moves quickly, Envoy AI Gateway targets a regular monthly release cadence (at least one release per calendar month) cut from the main branch, depending on the feature set and stability of the main branch at the time of the release.

Envoy AI Gateway is built on top of Envoy Gateway, so there is a notion of a "supported version" of Envoy Gateway that is compatible with Envoy AI Gateway.
Usually, the main branch of Envoy AI Gateway is tested against both the latest stable version of Envoy Gateway and its main branch.
However, Envoy AI Gateway may occasionally pick up critical new capabilities that are only available in the main branch of Envoy Gateway.

> In such cases, if a monthly release is based on an Envoy Gateway main commit (and not an upstream stable tagged release), the release notes will contain a clear WARNING stating this dependency.

Author: ```suggestion
In such cases, if a monthly release is based on the Envoy Gateway main branch (and not a stable tagged release), our release notes will contain a clear WARNING stating this dependency.
```
